console/console-init/ui/src/modules/connection-detail/components/ConnectionLinksToggleGroup/ConnectionLinksToggleGroup.tsx

```diff
@@ -210,7 +217,11 @@ const ConnectionLinksToggleGroup: React.FunctionComponent<IConnectionLinksToggle
         <>
           <FilterIcon />
           {checkIsFilterApplied() && (
-            <Badge key={1} isRead>
+            <Badge
+              id="connection-links-togglegrp-total-records-badge"
```

Comment:
Instead of using the 'badge' in the id, it would be better to name it on a functionality that this element refers to.